pkg/apis/application/v1alpha1/types_test.go

```diff
@@ -2619,6 +2619,7 @@ func TestGetCAPath(t *testing.T) {
 		"oci://bar.example.com",
 		"bar.example.com",
 		"ssh://foo.example.com",
+		"git@example.com/organization/reponame.git",
```

Comment:
Fwiw, this is not a proper git url.

It should be:
```
git@example.com:organization/reponame.git
```

Reply:
Thank you for your review.

I fixed it by https://github.com/argoproj/argo-cd/pull/9836/commits/ed7195cb10199a52cd34d8b1ccf9504973b15a32.